### DISCUSSION TITLE
Swag?

### DISCUSSION THREAD

**Discussion by reeceatkinson** [community/community#17699] [GitHub Education]
Is it possible to get GitHub Swag as a student for free?

**Comment by mlarasusan** [comment]
Hey Reece😃, you can definitely get swag for free as a student by participating in events and campaigns GitHub Education promotes, or if you become a Campus Expert for example. Something else you can do is contact your nearest GitHub Campus Expert to learn about upcoming events in your area that may have swag. Good luck! 😃

**Reply by binaryshrey** [reply]
One QQ:
Will the swags for GitHub grads, be shipped virtually this year?
As it states - Your PR must successfully merge into the repository and only the first 7,500 merged PRs will receive cards` in the mail`

.

**Reply by mlarasusan** [reply]
The cards will be physically sent by mail, to the students' addresses. Are you participating? [🤩](https://emojipedia.org/star-struck/)

**Reply by ZCyueternal** [reply]
when will the swag be sent to us...? or have github already mailed them to us ?
I haven't received them

**Reply by Singh212dab** [reply]
I participated but didn't get the swag and also getting error like "sorry the form cannot be committed"and also the names visible are upto "P" alphabet

**Reply by clonalejandro** [reply]
> when will the swag be sent to us...? or have github already mailed them to us ?
> I haven't received them

Me too :(..
